### ISSUE TITLE
Latest version no longer compatible with Node 12

### ISSUE THREAD

**Issue by robatwilliams** [jeffbski/wait-on#136]
In the past couple of days, this seems to have stopped working on Node 12 (v12.22.12):

```
npx wait-on http://localhost:8080
npx: installed 20 in 4.549s
Unexpected token '.'
```

I see this package hasn't been released in that period, so presumably something has changed in one of the dependencies (direct or transitive).

I'm aware that Node 12 (LTS) is end of life (https://endoflife.date/nodejs), however the readme of this package states it's supported. This issue arose in an old CI job that was using Node 12 and `npx wait-on` via https://github.com/cypress-io/circleci-orb

**Comment by robatwilliams** [comment]
https://github.com/hapijs/joi 17.8 was released yesterday and includes bumping up some of its dependencies

**Comment by zulaica** [comment]
According to [this comment](https://github.com/hapijs/joi/issues/2594#issuecomment-816481640) support for Node 12 was dropped in `joi` v17.

**Comment by robatwilliams** [comment]
As per the maintainer's response on joi#2914, they reverted 17.x (published 17.3 with the previous changes removed) and will release the changes as 18. This project specifies `^17.7.0` so won't pick that up automatically, so should now be fine.

**Comment by jeffbski** [comment]
Thanks for drawing my attention to this. wait-on is only able to support the active and maintained versions of Node.js due to dependencies and keeping up with security vulnerabilities. I have updated the README, github workflow, and package.json accordingly. Released 9.0.0 to draw attention to the reduction in Node.js version support. https://github.com/jeffbski/wait-on/releases/tag/v9.0.0
